workflows/VGP-assembly-v2/Scaffolding-HiC-VGP8/README.md

```diff
@@ -0,0 +1,20 @@
+# Scaffolding HiC
+
+Scaffolding using HiC data with YAHS
```

Comment:
Can you write a short sentence on what one would do before running this workflow and what you would do with the outputs ?